cli/cmd/testdata/inject_emojivoto_deployment_empty_resources.golden.yml

```diff
@@ -1,163 +0,0 @@
-apiVersion: apps/v1
```

Comment:
The reason why this file is expected to be empty is that if we pass a list of configs (delimited by `---`), and even if any one of the specs is non-compliant, `inject` does not output the remaining configs with the inject annotation present (that are expected to be piped to `kubectl`)
If this is non-trivial or should be regarded as unexpected behaviour, we may have to update the way `transformInput()` and `processYAML()` work, so that they may output the configs that were successfully annotated and show an error for the rest

Reply:
IMO, I don't think its non-trivial, As users would mostly group together resources in a file, and pass it to inject. Better to be pass or fail, so that users can correct rather than leaving in a confused install state. (even if we show errors for the rest)